data/charwidths.jl

```diff
@@ -113,7 +113,9 @@ for line in readlines(open("EastAsianWidth.txt"))
     for c in charstart:charend
         if width=="W" || width=="F" # wide or full
             CharWidths[c]=2
-        elseif width=="Na"|| width=="H" # narrow or half
+        elseif width=="Na" || width=="H" # narrow or half
+            CharWidths[c]=1
+        elseif width=="N" && haskey(CharWidths, c)
```

Comment:
Yeah, I had that, but it didn't actually make a difference to the output. I can add it back, if you'd like.